Vineyard/Vineyard/ProfileView.swift

```diff
@@ -8,11 +8,82 @@
 import SwiftUI
 
 struct ProfileView: View {
+    @State private var viewModel = ProfileViewModel()
```

Comment:
If you are using `@State`, you should use the `@Observable` macro for the class. If you wanna go ahead with the old `ObservableObject` protocol, this should be a `@StateObject`.

We can take care of this later in another issue/PR. Can you create one for that?

Reply:
I will create one for that